src/librustc_builtin_macros/global_allocator.rs

```diff
@@ -80,11 +80,7 @@ impl AllocFnFactory<'_, '_> {
     }
 
     fn call_allocator(&self, method: &str, mut args: Vec<P<Expr>>) -> P<Expr> {
```

Comment:
You could also change `AllocatorMethod::name` to be a `Symbol`

Reply:
True, I will do that.